addons/mrp/report/report_stock_reception.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-    <!-- Reception Report Labels -->
-    <template id="report_reception_report_label_mrp">
```

Comment:
I guess we should ony remove in master to have a migration script. (idem for the ir.action.report)